src/storage/mod.rs

```diff
@@ -5698,4 +5700,154 @@ mod tests {
         let res = rx.recv().unwrap().unwrap();
         assert_eq!(res.min_commit_ts, 1001.into());
     }
+
+    // this test shows that the scheduler take `response_policy` in `WriteResult` serious,
```

Comment:
This test should be in `scheduler.rs`, but I haven't found a good way to mock a scheduler.